### ISSUE TITLE
Introduce ingress transactions to reduce orphan files opportunities

### ISSUE THREAD

**Issue by raphaelvp** [qe-collaborative-services/1115-hub#112]
Orphan files can occur when ingress of multiple batches can cross boundaries. Introduce a feature to move all ingress files into `egress/<session-id>/.ingress-tx` before beginning consumption of files.

**Comment by alan-francis** [comment]
The following error is logged to the observation log while no files are available in the ingress folder.

```
error: Uncaught (in promise) NotFound: No such file or directory (os error 2): readdir '/SFTP/healthelink/egress/f70028e9-934d-4341-9392-f296ede5261b/.ingress-tx'
      for await (const entry of Deno.readDir(watchPath.rootPath)) {
                       ^
    at async Object.[Symbol.asyncIterator] (ext:deno_fs/30_fs.js:222:19)
    at async Module.ingestWatchedFs (https://raw.githubusercontent.com/netspective-labs/sql-aide/v0.13.15/pattern/orchestration/ingest-fs.ts:61:24)
    at async Command.actionHandler (file:///app/1115-hub/src/ahc-hrsn-elt/screening/orchctl.ts:287:7)
    at async Command.execute (https://deno.land/x/cliffy@v1.0.0-rc.3/command/command.ts:1948:7)
    at async Command.parseCommand (https://deno.land/x/cliffy@v1.0.0-rc.3/command/command.ts:1780:14)
    at async file:///app/1115-hub/src/ahc-hrsn-elt/screening/orchctl.ts:196:1
```

**Comment by shah** [comment]
Released in `v0.13.0`.
